docs/references/func.md

```diff
@@ -1,18 +1,8 @@
-# SQL
+# Func
 
 Use SQL functions to operate on the underlying database storing the chain data. Useful
```

Comment:
`SQL` should be replaced with something (not sure what).

Also need to remove `Import these fucntions from ...` statement.

Reply:
let's just use Functions for now